packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -198,13 +198,13 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
   Future<void> processSyncRequests(
       {bool respectSyncRequestQueueSizeAndRequestTriggerDuration =
           true}) async {
-    final syncProgress = SyncProgress()..syncStatus = SyncStatus.started;
-    syncProgress.startedAt = DateTime.now().toUtc();
     _logger.finest('in _processSyncRequests');
     if (_syncInProgress) {
       _logger.finer('**** another sync in progress');
-      syncProgress.message = 'another sync in progress';
-      _informSyncProgress(syncProgress);
+      _informSyncProgress(SyncProgress()
+        ..syncStatus = SyncStatus.started
+        ..startedAt = DateTime.now().toUtc()
+        ..message = 'another sync in progress');
```

Comment:
Remove a bug vector by creating a SyncProgress each time when informing about sync progress, rather than reusing the same syncProgress.